Готова 8 проектная работа

nastyaprana opened the pull request:
Отправляю на ревью

nastyaprana committed:
Сделала 8 проект

nastyaprana committed:
Внесла правки после тестов

nastyaprana committed:
Заново перезалила тесты

Была ошибка   "-e 1. Только SVG с сердцем внутри карточки вставлены инлайново (необходимо для анимации), остальные — через symbol + use."
Но все svg вставлены верно, все на своих местах.
Заново перезалила тесты с 8 проекта..

nastyaprana committed:
Обновила тесты вручную

nastyaprana committed:
Добавила @supports text-stroke/webkit-text-stroke

AliaksandrHrychonak commented on styles/style.css:
```diff
@@ -15,6 +15,450 @@
+  }
+}
+
+.image-2 {
```
Надо исправить

Используйте наименования классов по типу применяемого фильтра, например `filter-blur`. Фильтры в проекте должны быть универсальными и переиспользуемыми. Они не должны быть привязаны к конкретному порядку изображений или типу их содержания.

Такой подход позволяет легко понять назначение фильтра и применять его к различным изображениям. Это также упростит добавление новых фильтров или изменение существующих в будущем.

AliaksandrHrychonak commented on index.html:
```diff
@@ -1,46 +1,428 @@
+            <use href="./svg/floppy.svg#floppy" />
+          </svg>
+        </div>
+        <h2 class="dialog-card__text">Вставьте <br> дискету, чтобы продолжить</h2>
```
Не нужно использовать `br` для переноса строк внутри одного текстового элемента - для этого обычно ограничивают максимальную ширину элемента и тогда он перенесется на новую строку сам. 
 ```br``` использовать в таких случаях плохо потому что если текст в элементе может изменится - перенос строки перетрется, или будет работать некорректно.

Данный тег является мощным инструментом для переноса, его использование следует свести к минимуму.

AliaksandrHrychonak commented on index.html:
```diff
@@ -1,46 +1,428 @@
+      </div>
+      <div>
+        <button class="form-button button" type="button" onclick="window['dialog-id'].close()">
+          <span class="form-button__text dialog-card__text">ок</span>
```
Переиспользуйте  `button__text`  для сокращения дублирования одинакового кода

AliaksandrHrychonak reviewed (commented):
Необходимо развернуть общий комментарий ⬇️

Добрый день. Работа выполнена на хорошем уровне! 

Отлично:

* Размечена вся структура
* Используете семантичные теги для разметки.
* аккуратно оформленный код
* все комментарии "Отлично"

Надо исправить:

* Все комментарии с пометкой "надо исправить"
- Границы не по заданию https://disk.yandex.com/i/-SfPfoKvZ3Ozag
* В проекте есть нарушения в отображении цветовых палитр в темной и светлой теме, цветовые палитры необходимо полностью проконтролировать
    * Например https://disk.yandex.com/i/-ROB1BHRmas1ig цвета не по макету
    * https://disk.yandex.com/i/oWrQ1srWIHxnDw 
    * авто тёмная тема не настроена https://disk.yandex.com/i/Ho3x5xnlo_p-Rw
* https://disk.yandex.com/i/DBvFQu3-g_9XHA При снятии/при наведении на кнопки с текстом текст не должен пропадать, переход цвета должен происходить плавно в обе стороны, используйте `mix-blend-mode` для смены цвета
* В проекте есть нарушения в отображении по макету
    * **На 375px не по макету:**
      * https://disk.yandex.com/i/Mh1AZ8zHBR0E-Q
      * https://disk.yandex.com/i/zjnAL2CsgcSEuA
    * Для более точного контроля отображения вёрстки можно воспользоваться плагином `PerfectPixel` для `Chrome`, можно в фигме выделить весь макет, экспортировать его картинкой и наложить на вёрстку в браузере. Плагин позволяет наложить изображение на вёрстку в браузере, тем самым можно слегкостью подправить недочёты по отображению. 


Можно лучше:

* Все комментарии "Можно лучше" в коде.
* В качестве альтернативного шрифта для `PressStart2P` по типу засечек можно использовать `fantasy`

Необходимо исправить все "надо исправить". Исправление "можно лучше" остаются на ваше усмотрение.

nastyaprana committed:
Внесла правки после ревью

Исправила нарушения в отображении в разных цветовых палитрах.
Исправила нарушения в отображении по макету.
Сделала корректный переход кнопок в темной теме.
Внесла правки согласно комментариям.
Также настроила авто темную тему и доработала favicon для авто темной темы.

nastyaprana committed:
Поправила стиль кнопки в модальном окне

AliaksandrHrychonak commented on styles/style.css:
```diff
@@ -15,6 +15,443 @@
+}
+
+.dialog[open]::backdrop {
+  background-color: rgb(0 0 0 / 0.7);
```
По макету прозрачность 0.75

AliaksandrHrychonak reviewed (commented):
Добрый день, отличные исправления, проект принят!
